relay-general/src/protocol/measurements.rs

```diff
@@ -0,0 +1,127 @@
+use regex::Regex;
+
+use crate::types::{Annotated, Error, FromValue, Object, Value};
+
+lazy_static::lazy_static! {
+    static ref MEASUREMENT_NAME: Regex = Regex::new("^[a-z0-9-._]+$").unwrap();
```

Comment:
This is also defined in `protocol/contexts.rs`, I'd avoid the duplication.

Reply:
now removed in https://github.com/getsentry/relay/pull/724/commits/ada8e4ff054112d1150192159a38891f2533de88

Reply:
Can we replace this regex with some function? Regexes turn out to be pretty expensive and we're trying to remove them across the code base where we can.

Reply:
I'm open to suggestions.

I'm unsure what the alternative is.

Reply:
The alternative would be to create a function that returns `false` if the string contains any invalid character.

Reply:
Should be addressed in https://github.com/getsentry/relay/pull/724/commits/386facfb96d6aa88abbe0e43b6f4bd47b8ae45c8